fix for last open items at https://github.com/OWASP/owasp-masvs/issues/203 (#1479)
OWASP/owasp-mstg: Document/0x05h-Testing-Platform-Interaction.md

</manifest>
```

> We recommend using a the reverse-domain annotation when registering a permission, like in the example above (e.g. `com.domain.application.permission`) in order to avoid collisions with other applications.

Reviewer: ```suggestion
We recommend using a reverse-domain annotation when registering a permission, as in the example above (e.g. `com.domain.application.permission`) in order to avoid collisions with other applications.
```

This allows the WebView to interpret JavaScript. It should be enabled only if necessary to reduce the attack surface to the app. If JavaScript is necessary, you should make sure that

> - The communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission

Reviewer 2: ```suggestion
- The communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission.
```

- JavaScript and HTML are loaded locally, from within the app data directory or from trusted web servers only.

> - The user cannot define which sources to load by means of loading different resources based on a user provided input.

Reviewer 2: I tried a different re-phrasing for this based on your additions below. 

```suggestion
- The WebView's URI cannot be manipulated by the user in order to load different resources than the ones provided by the app.
```

Author: It can be more than that. Let's just resolve

Author: It can be more than that. Let's just resolve

> - the communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission

Reviewer: ```suggestion
- The communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission
```

- JavaScript and HTML are loaded locally, from within the app data directory or from trusted web servers only.

> - The user cannot define which sources to load by means of loading different resources based on a user provided input.

Reviewer: ```suggestion
- It is not possible to navigate the WebView to a specified URL through app-to-app communication without properly validating the URL.
- It is not possible to navigate to untrusted sources by means of interacting with the WebView.
```
Modified it somewhat. Allowing a user to choose which page to load seems ok at first glance.
Also added one other scenario. This one actually has a preventive measure, so maybe that's already tackled somewhere else?

Author: This does not make sense: there are many type of ways how a user can load that resource: IPC, wrong input within the app itself, so closing this one :)
